preprocessing.py

```diff
@@ -19,9 +19,10 @@
 def process_raw_data(use_gpu, force_pre_processing_overwrite=True):
     print("Starting pre-processing of raw data...")
     input_files = glob.glob("data/raw/*")
+    print(input_files)
     input_files_filtered = filter_input_files(input_files)
     for file_path in input_files_filtered:
-        filename = file_path.split('/')[-1]
+        filename = file_path.split('\\')[-1]
```

Comment:
This looks Windows specific - maybe check what Operating System you're on and the split accordingly? (we don't want to lose compatibility on Linux/Mac)